nearcore/src/lib.rs

```diff
@@ -400,7 +393,7 @@ pub fn start_with_config_and_synchronization(
     // `ClientActor` gets dropped.
     shutdown_signal: Option<oneshot::Sender<()>>,
 ) -> Result<NearNode, anyhow::Error> {
```

Comment:
```suggestion
) -> anyhow::Result<NearNode> {
```

If we are touching the code around anyway?